xcorrmap.m

```diff
@@ -0,0 +1,50 @@
+function [map, mean_spikes] = xcorrmap(data,sel,show)
```

Comment:
Why are there three xcorrmap??? Which one is the good one? hahaha

Reply:
this is hahaha